satellite/peer.go

```diff
@@ -489,7 +491,18 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, revocationDB exten
 		}
 
 		// setup audit 2.0
-		peer.Audit.ReservoirService = audit.NewReservoirService(peer.Log.Named("reservoir service"),
+		peer.Audit.Queue = &audit.Queue{}
+
+		peer.Audit.Worker, err = audit.NewWorker(peer.Log.Named("audit service 2"),
```

Comment:
Is there any plan to remove the `2` from the logger at some point in the future?